docs/DeveloperGuide.md

```diff
@@ -589,6 +591,16 @@ Duplicate names)
 
 ---
 
+## **Appendix: Planned Enhancements**
+
+Team member count: 5
+
+1. **Implement commands to add/edit assignment details:** Currently, one has to edit the `kontacts.json` file manually to add or remove assignment details. This could be an issue as the information keyed in may be incorrect. We are planning to create commands which can add/edit assignment details (such as assignment name `Ex04` and its respective `maxGrade` fields) to the `assignment.json` file. This is to make the adding/editing of new assignments easier and prevent wrong information being entered.
+2. **Include validation for telegram ID:** Currently, KonTActs does not chack for the telegram ID's length despite it not being a valid username in telegram. In future updates, we are planning to provide proper telegram ID validation to ensure that there can not be two people with the same telegram IDs.
```

Comment:
I think we can remove the check for duplicates since it can be possible that two students may want to use the same telegram id. Perhaps just checking for proper telegram id format will do?

Reply:
Alright got it!